pkg/features/kube_features.go

```diff
@@ -1029,6 +1037,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	SizeMemoryBackedVolumes: {Default: true, PreRelease: featuregate.Beta},
 
+	StableLoadBalancerNodeSet: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
gce is the main stakeholder, I would say the only one of this change, I don't think we should rollout this change enable by default @thockin ... there is not testing at all in the fleet for this

Reply:
I thought I commented in the KEP about this but I can't find the comment, I see that it merged with beta as default 🤷 , we are going YOLO then

Reply:
> gce is the main stakeholder (the only one) of this change 

No, you're confusing this with "KEP 2", the one changing Kube-proxy's `healthz` to also inspect the Node state - probably the `unschedulable` field - when determining if it should be healthy or not. That impacts GCP because GCP is the only one targeting Kube-proxy's healthz for eTP:Cluster services. 

This applies to all cloud providers equally.